kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/producer/KafkaEventPublisher.java

```diff
@@ -69,6 +70,7 @@ public static <K, V> Builder<K, V> builder() {
      */
     protected KafkaEventPublisher(Builder<K, V> builder) {
         builder.validate();
+        this.processingGroup = builder.processingGroup != null ? builder.processingGroup : DEFAULT_PROCESSING_GROUP;
```

Comment:
I think it's nice for consistency to initialize the `processingGroup` with the default value in the builder. Then we can remove this.

Reply:
Done!